test/mockContracts/EntitiesTransferManagerMock.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity 0.4.24;
```

Comment:
why you need the mock contract? isn't it the same functionality as the `EntitiesTransferManager` itself?

Reply:
right! I needed it  to tess fucntion overloading (two `addRule` fucntions in the contract). Now I can use function overloading in truffle v5 without the mock contract.